src/main/java/com/cburch/logisim/util/LineBuffer.java

```diff
@@ -167,14 +167,16 @@ protected LineBuffer addDefaultPairs() {
   public LineBuffer addHdlPairs() {
     return pair("assign", Hdl.assignPreamble())
         .pair("=", Hdl.assignOperator())
+        .pair("==", Hdl.equalOperator())
+        .pair("!=", Hdl.notEqualOperator())
         .pair("or", Hdl.orOperator())
         .pair("and", Hdl.andOperator())
-        .pair("not", Hdl.notOperator())
         .pair("xor", Hdl.xorOperator())
-        .pair("bracketOpen", Hdl.bracketOpen())
-        .pair("bracketClose", Hdl.bracketClose())
+        .pair("not", Hdl.notOperator())
         .pair("<", Hdl.bracketOpen())
         .pair(">", Hdl.bracketClose())
+        .pair("else", Hdl.elseStatement())
+        .pair("endif", Hdl.endIf())
```

Comment:
If I am not mistaken, `endif` is not a keyword neither in VHDL nor in (System)Verilog. Is this therefore maybe a typo?

Reply:
It is not,
neither VHDL nor Verilog feature this keyword, but both have keywords to end an if statement.
(VHDL: `END IF;` & Verilog: `end`)
`endif` was the first thing I could think of that described what this code is doing.